tests/integrations/examples/test_driftbif_simulation.py

```diff
@@ -63,6 +64,18 @@ def test_dimensionality(self):
         self.assertEqual(v.shape, (Nt, 3),
                          'The returned vector should reflect the dimension of the initial condition.')
 
+    def test_relevant_feature_extraction(self):
```

Comment:
Why is this test catching the bug?

Reply:
Because string identifiers are not sorted in the same order as integers, which makes y and the extracted features having a different order.
Any example where y is not order "normally" would be a good example, but as we do not have any other example with string identifiers I thought it might be good to add one anyways